posts/demos/_posts/2013-10-06-controls-customization.html

```diff
@@ -3,6 +3,15 @@
 title: Controls customization
 ---
 
+<style> 
+#setControlVisible {
```

Comment:
Btw, I suggest to never use id's in CSS. Check out http://csswizardry.com/2012/11/code-smells-in-css/#ids